ui/components/grant.js

```diff
@@ -21,8 +21,9 @@ function User ({ id }) {
 
 export default function ({ id }) {
   const { data: destination } = useSWR(`/api/destinations/${id}`)
-  const { data: { items: list } = {} } = useSWR(() => `/api/grants?resource=${destination.name}`)
+  const { data: { items } = {} } = useSWR(() => `/api/grants?resource=${destination.name}`)
   const { mutate } = useSWRConfig()
+  const list = items?.filter(item => item.user !== undefined)
```

Comment:
Taking away the check against `undefined`, while seeming ambiguous, will catch other potential cases like `null`:

Checking `!!item.user` works too

```suggestion
  const list = items?.filter(item => item.user)
```